todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -122,65 +188,161 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         super.onCreateOptionsMenu(menu, inflater);
     }
 
-    @Override
-    public void setLoadingIndicator(boolean active) {
-        if (active) {
-            mDetailTitle.setText("");
-            mDetailDescription.setText(getString(R.string.loading));
+    private void showTask(@NonNull Task task) {
```

Comment:
as i said just before V becomes way too complex. You said it also V(actually the binder) observes for data on the VM and updates itself(Actually the binder job as well) syncing of the data should be a V problem the idea is that V is super simple IMO :) at the end we don't want many GUI inst it?